src/NuGet.Clients/VisualStudio.Facade/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "NuGet.Common": "3.6.0-*",
```

Comment:
Until now we didn't need to access Nuget.Common package in facade project, this is the new dependency.